pygenometracks/tests/test_data/bigwig.ini

```diff
@@ -0,0 +1,70 @@
+
+[test bigwig lines]
+file = bigwig2_X_2.5e6_3.5e6.bw
```

Comment:
this file does not need to be included, it is generated automatically, isn't it?

Reply:
no, is not generated automatically. They are part of the test_data